src/Query/Conditions/BetweenColumnsCondition.php

```diff
@@ -36,7 +36,7 @@
  */
 class BetweenColumnsCondition implements BetweenColumnsConditionInterface
 {
-    public function __construct(
+    final public function __construct(
```

Comment:
Why? Class not final, how about inheritance?

Reply:
The psalm marks a constructor inconsistency error and recommends making it final.